src/openapi/core/layer.yaml

```diff
@@ -0,0 +1,38 @@
+openapi: '3.0.1'
+info:
+  title: Layer properties definitions
```

Comment:
its feels like we have duplication schemas as we defined it already within "mc-models" - lets talk about this one.

https://github.com/MapColonies/mc-models/blob/68576fc1409d5f0dbbed2c127699be8b7dd1a795/src/yaml/ingestionTrigger/metadata/rasterLayerMetadata.yaml#L29